Gemfile.lock

```diff
@@ -0,0 +1,39 @@
+GEM
```

Comment:
Should just be a matter of moving the `clone` into the `install` steps and adding a couple `cp` commands after the `clone`.